homeassistant/loader.py

```diff
@@ -97,10 +97,10 @@ class BlockedIntegration:
     "dreame_vacuum": BlockedIntegration(
         AwesomeVersion("1.0.4"), "crashes Home Assistant"
     ),
-    # Added in 2024.5.5 because of
-    # https://github.com/sh00t2kill/dolphin-robot/issues/185
+    # Added in 2024.8.0 because of
+    # https://github.com/sh00t2kill/dolphin-robot/issues/221
     "mydolphin_plus": BlockedIntegration(
-        AwesomeVersion("1.0.13"), "crashes Home Assistant"
+        AwesomeVersion("1.0.17"), "thousands of requests Maytronics login API per user"
```

Comment:
Honestly, this is not designed for being an upgrade notice system.

We use this block list to block integrations that break Home Assistant. As far as I can see, that isn't the case here?

../Frenck

Reply:
Yes, agree, maybe it should be added as an option for developers to block older versions because something got broken with API or more like in the use case I have here,
If it should be developed within core as separated feature (and you think is good feature to have), will work on it 

Thanks

Reply:
> ybe it should be added as an option for developers to block older versions because something got broken with API or more like in the use case I have here,

If you feel like your integration needs something like this, you are free to build that in your own integration. Third-party integrations are not our responsibility.

Reply:
If that's how you see it, I think PR can be rejected,
Thought about it as something to protection in general for components that major bug found in it, regardless if it has an impact on HA or on the 3rd party